stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -1276,24 +1276,25 @@ internal class ChannelController(
          * before what we've last pushed to the UI. We want to ignore this, as it will cause an unread state
          * to show in the channel list.
          */
```

Comment:
Even if there is no incoming user read for current user we should still update the `_reads.value` below.